contracts/GuestList.vy

```diff
@@ -55,13 +90,31 @@ def set_bouncer(new_bouncer: address):
     self.bouncer = new_bouncer
 
 
+@view
+@internal
+def _yfi_in_makerdao(user: address) -> uint256:
+    proxy: address = self.proxy_registry.proxies(user)
+    cdp: uint256 = self.cdp_manager.first(proxy)
+    urn: address = ZERO_ADDRESS
+    total: uint256 = 0
+    for i in range(100):
```

Comment:
why are you limiting to 100?

Reply:
Vyper only allows literals here, so I can't do an unbounded loop. 100 sounded like well over how many CDPs a single user can have open, Maker doesn't even have that many collateral types and it likely won't change in the next 30 days.

Reply:
agree, for a person that does have many CDPs the deposit into the vault will be gas costly